src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -459,6 +460,11 @@ export default class FilingHistoryList extends Vue {
     return filing.availableOnPaperOnly || filing.isTypeStaff || filing.documentsLink
   }
 
+  /** Whether this filing is a Court Order. */
+  isCourtOrder (filing): boolean {
+    return this.isTypeCourtOrder({ name: filing.name })
+  }
+
```

Comment:
Rather than creating a new computed property, what do you think of calling `isTypeCourtOrder()` from line 315 above?

Reply:
Alternatively, what do you think of creating a Vuex getter?  It looks like we need a method that determines if a filing is a court order in multiple places.

Reply:
There's a mixin with helper methods like this. I think this should be moved there,

Reply:
isTypeCourtOrder() is already in the 'EnumMixin' which works too. I think using it should be okay too.

Reply:
LGTM. Jonathan, please resolve this conversation if you are OK with it.